desc/objectives/_profiles.py

```diff
@@ -129,18 +125,25 @@ def build(self, eq=None, use_jit=True, verbose=1):
             scales = compute_scaling_factors(eq)
             self._normalization = scales["p"]
 
-        super().build(eq=eq, use_jit=use_jit, verbose=verbose)
+        super().build(things=eq, use_jit=use_jit, verbose=verbose)
 
-    def compute(self, *args, **kwargs):
-        """Compute plasma pressure.
+    def compute(self, params, constants=None):
+        """Compute the quantity.
+
+        Parameters
+        ----------
+        params : dict
+            Dictionary of equilibrium degrees of freedom, eg Equilibrium.params_dict
+        constants : dict
+            Dictionary of constant data, eg transforms, profiles etc. Defaults to
+            self.constants
 
         Returns
         -------
-        pressure : ndarray
-            Plasma pressure at specified points.
+        f : ndarray
+            Computed quantity.
```

Comment:
is this not the pressure? why change the docstring?

Reply:
I just copy and pasted the docstring from the other objectives